docs/data/material/components/tabs/NavTabs.js

```diff
@@ -27,11 +28,17 @@ function LinkTab(props) {
           event.preventDefault();
         }
       }}
+      aria-current={props.selected && 'page'}
+      role="navigation"
```

Comment:
```suggestion
```

`role="navigation"` should be at Tabs element, not Tab.

Reply:
Fixed.